app/code/Magento/Eav/Model/Entity/Attribute/AbstractAttribute.php

```diff
@@ -119,6 +119,11 @@
      */
     protected $dataObjectHelper;
 
+    /**
+     * @var \Magento\Eav\Model\Entity\Attribute\FrontendLabelFactory
+     */
+    protected $frontendLabelFactory;
```

Comment:
Please, make this property private

Reply:
Not sure why this property should be private. Could you please give me more details about that?

Thanks!

Reply:
Please, see more details in [Magento Technical Guidelines](http://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html#2-class-design) 

> 2.7. All non-public properties and methods SHOULD be private.

Reply:
Thanks for pointing that out, I didn't know all those rules!